package.json

```diff
@@ -15,9 +15,9 @@
     "copy:assets": "cpr assets build/assets -d",
     "copy:scss": "cpr scss build/scss -o",
     "build:style-dictionary": "node style-dictionary.js && npm run copy:scss",
-    "build:patternlab-css": "sass source/css/db-ui-base.scss:out/css/db-ui-base.css --style=compressed --embed-sources --load-path=node_modules",
+    "build:patternlab-css": "sass source/css/patternlab.scss:out/css/patternlab.css --style=compressed --embed-sources --load-path=node_modules",
```

Comment:
@annsch why not call it `documentation.scss` or something similar instead?

Reply:
@mfranzke this branch was merged without duscussing your feedback, sry. I'll open an issue for this.